composer.json

```diff
@@ -15,8 +15,8 @@
         "symfony/finder": "~2.7|~3.1",
         "symfony/console": "~2.7|~3.1",
         "symfony/intl": "~2.7|~3.1",
-        "symfony/templating": "~2.7|~3.1",
-        "symfony/translation":  "~2.7|~3.1"
+        "symfony/translation":  "~2.7|~3.1",
+        "twig/twig": "~1.0|~2.0"
```

Comment:
I suggesting depending on TwigBundle instead (as this bundle actually depends on services defined in the bundle, not just on Twig)

Reply:
Done :+1: